aday2dream/src/main/kotlin/com/aday2dream/aday2dream/service/AccountService.kt

```diff
@@ -23,11 +31,11 @@ class AccountService(@Autowired
         )
     }
 
-    private fun mapToEntity(userDto: AccountDto): Account {
+    private fun mapToEntity(userDto: AccountDto, hashedPassword: String): Account {
```

Comment:
you can use mapstruct -> https://www.baeldung.com/kotlin/mapstruct-data-classes
it will save you time + get rid of such boilerplate code